graylog2-web-interface/src/pages/LoggedInPage.jsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import AppRouter from 'routing/AppRouter';
 import CurrentUserPreferencesProvider from '../contexts/CurrentUserPreferencesProvider';
+import CustomizationProvider from '../contexts/CustomizationProvider';
```

Comment:
This probably will need to wrap around the ThemeProvider stuff so that in the future we can reference the CustomizationProvider inside of `src/theme/GraylogThemeProvider.jsx` and get any modified color values.

Reply:
Should we do it now or later?

What do you suggest exactly should I do?

Reply:
I do have some suggestions, but may require some back-and-forth to make sure all our needs are accounted for. Let's plan a time to pair up for 30 minutes and discuss this!